packages/myst-frontmatter/tests/authors.yml

```diff
@@ -50,50 +70,167 @@ cases:
+          nameParsed:
+            display: Just A. Name
+            given: Just A.
+            family: Name
           email: example@example.com
           corresponding: false
   - title: Respect corresponding flag
     raw:
       authors:
-        - name: Someone
+        - name: Just A. Name
           email: example@example.com
-        - name: Next
+        - name: A. Nother Name
           email: example@example.com
           corresponding: true
     normalized:
       authors:
-        - name: Someone
+        - name: Just A. Name
+          nameParsed:
+            display: Just A. Name
+            given: Just A.
+            family: Name
           email: example@example.com
-        - name: Next
+        - name: A. Nother Name
+          nameParsed:
+            display: A. Nother Name
+            given: A. Nother
+            family: Name
           email: example@example.com
           corresponding: true
   - title: Warn if author has no name
     raw:
       authors:
-        - name: Someone
+        - name: Just A. Name
           email: example@example.com
         - email: example@example.com
           corresponding: true
     normalized:
       authors:
-        - name: Someone
+        - name: Just A. Name
+          nameParsed:
+            display: Just A. Name
+            given: Just A.
+            family: Name
           email: example@example.com
         - email: example@example.com
           corresponding: true
     warnings: 1
+  - title: Warn if author has no family name
+    raw:
+      authors:
+        - name:
+            given: Someone
+    normalized:
+      authors:
+        - name: ', Someone'
+          nameParsed:
+            display: ', Someone'
+            given: Someone
+    warnings: 1
+  - title: Warn if author has no given name
+    raw:
+      authors:
+        - name: Someone
+    normalized:
+      authors:
+        - name: 'Someone'
+          nameParsed:
+            display: 'Someone'
+            family: Someone
+    warnings: 1
+  - title: Parsed name is maintained
+    raw:
+      authors:
+        - name:
+            display: Just A. Name
+            given: Little
+            particle: von
+            family: Junior
+            suffix: Jr.
+    normalized:
+      authors:
+        - name: Just A. Name
+          nameParsed:
+            display: Just A. Name
+            given: Little
+            particle: von
+            family: Junior
+            suffix: Jr.
+  - title: Name object with lots of commas warns
+    raw:
+      authors:
+        - name:
+            display: Commas, here, are, great,
+            given: Commas,
+            particle: here,
+            family: are,
+            suffix: warnings,
+    normalized:
+      authors:
+        - name: Commas, here, are, great,
+          nameParsed:
+            display: Commas, here, are, great,
+            given: Commas,
+            particle: here,
+            family: are,
+            suffix: warnings,
```

Comment:
where do warnings come from?

Reply:
we are warning about the commas and there are 4 instances?

Reply:
Yes - The idea is if a comma ends up in any of the parsed parts, the name was maybe not parsed as expected.